tests/appsec/iast/source/test_kafka_key.py

```diff
@@ -0,0 +1,23 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import coverage, features, scenarios
+from .._test_iast_fixtures import BaseSourceTest, get_all_iast_events, get_iast_sources
+
+
+@coverage.basic
+@features.iast_source_kafka_key
+@scenarios.integrations
+class TestKafkaKey(BaseSourceTest):
+    """Verify that kafka message key is tainted"""
+
+    endpoint = "/iast/source/kafkakey/test"
+    requests_kwargs = [{"method": "GET"}]
+    source_type = "kafka.message.key"
+    source_names = ["key"]
```

Comment:
The name is hardcoded right?

Reply:
yes, I have removed them from the tests. As they are hardcoded they don't contribute much